README.md

```diff
@@ -57,6 +57,8 @@ Most developers begin by identifying an interesting sample based on a specific u
   This sample demonstrates how to create an Apigee Integrated portal and shows how to expose your API products to its catalog
 - [exposing-to-internet](exposing-to-internet) -
   This sample demonstrates how to expose an Apigee instance to the internet using a Google Cloud external HTTP(S) Load Balancer
+- [json-web-tokens](json-web-tokens) -
+  This sample demonstrates how to generate and verify JSON Web Tokens
```

Comment:
should we emphasize

`  This sample demonstrates how to generate and verify JSON Web Tokens using the out of the box Apigee JWT policies` ?